nav/support/correlate.py

```
@@ -1,701 +1,553 @@
+            nms_radius=nms_radius, logger=logger
+        )
+        logger.debug(f'Correlation pyramid level {lvl} results: '
+                     f'offset {res_lvl["offset"][0]*s:.3f}, {res_lvl["offset"][1]*s:.3}; '
+                     f'sigma_xy {res_lvl["sigma_xy"][0]*s:.3f}, {res_lvl["sigma_xy"][1]*s:.3}; '
+                     f'quality {res_lvl["quality"]:.3f}; '
+                     f'peak_val {res_lvl["peak_val"]:.3f}')
+
+        # Scale shift back to full res
+        level_shifts.append((res_lvl['offset'][0]*s, res_lvl['offset'][1]*s))
+
+    # Consistency: max deviation to last level’s shift
```

Ruff (0.14.4)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)